### PR TITLE
[4][cli] update:extensions:check

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #37643 .

### Summary of Changes
There is no need to show cahed result on cli

### Testing Instructions
see #38119

### Actual result BEFORE applying this Pull Request
[OK] There are no available updates

### Expected result AFTER applying this Pull Request
! [NOTE] There are available updates to apply

### CODE REVIEW

**Comment by HLeithner** [comment]
A new pull requests has been created automatically to convert this PR to the PSR-12 coding standard.\n The pr can be found at https://github.com/alikon/joomla-cms/pull/3

**Comment by alikon** [comment]
it's closed by you

**Comment by richard67** [comment]
I think @HLeithner is playing with the tools, and his above comment might have been created automatically.

**Comment by HLeithner** [comment]
> it's closed by you

yes sorry I used your PR by accident

File: libraries/src/Console/CheckUpdatesCommand.php [hunk 1]
```diff
@@ -9,7 +9,8 @@
 
 namespace Joomla\CMS\Console;
 
-use Joomla\CMS\Component\ComponentHelper;
+\defined('JPATH_PLATFORM') or die;
+
```
Comment (richard67) [inline libraries/src/Console/CheckUpdatesCommand.php]:
```suggestion
```
Was added by the psr12 conversion by mistake and now causes PHPCS check to fail in drone. Should be removed again.

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 5288d0daa80a36b4ed179644e802fc057c5198e7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38152">issues.joomla.org/tracker/joomla-cms/38152</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 5288d0daa80a36b4ed179644e802fc057c5198e7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38152">issues.joomla.org/tracker/joomla-cms/38152</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38152">issues.joomla.org/tracker/joomla-cms/38152</a>.</sub>

**Comment by roland-d** [comment]
Thank you

**Pull request merged**